### DISCUSSION TITLE
Is PWA studio support multiple website and multiple stores now?

### DISCUSSION THREAD

**Discussion by hkvega01** [magento/pwa-studio#3797] [Q&A]
anyone knows?

**Comment by hoangngo-calvin** [comment]
Yes, I tried to configure 1 BE with multiple websites. It seem to works

**Reply by hkvega01** [reply]
thanks for the reply, but i didn't find any tutorial on internet, do any documentation show how to do that?

**Reply by hoangngo-calvin** [reply]
in the .env file, you just specify the corresponding default store view code for each website: ```STORE_VIEW_CODE```
eg:
- Website 1: .env file configure STORE_VIEW_CODE=web_1_store_1_view_1
- Website 2: .env file configure STORE_VIEW_CODE=web_2_store_2_view_2

**Reply by hkvega01** [reply]
> 

seems like need to separate code base for pwa studio if running multiple sites

**Reply by FrancescoTerreni** [reply]
Are there any news?
We should be able to add different domains for storeview.
